src/styles/block/card.scss

```
@@ -0,0 +1,99 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
```

don't forget to create variables for all colors
```suggestion
  border: 1px solid ...;
```